api/src/main/java/org/openmrs/module/ModuleUtil.java

```diff
@@ -416,7 +416,7 @@ public static void checkRequiredVersion(String version, String versionRange) thr
 	 */
 	public static int compareVersion(String version, String value) {
 		try {
-			if (version == null || value == null) {
+			if (version == "1.8.4-SNAPSHOT" || value == "1.8.4") {
```

Comment:
@ODORA0 what were you trying to do here??

Reply:
I believe this change isn't necessary as well, guess discarding the change would be good @mozzy11

Reply:
ya is unnecesary